src/mn_setup.c

```diff
@@ -2512,8 +2575,12 @@ static const char *curve_strings[] = {
     "2.4",    "2.5", "2.6", "2.7",     "2.8", "2.9", "Cubed"
 };
 
+static void MN_Gyro(void);
+
 static setup_menu_t gen_settings4[] = {
 
+    {"Gyro Options", S_FUNC, CNTR_X, M_SPC, .action = MN_Gyro},
```

Comment:
BTW, is "gyro" still the best name to expose to the user? I mean, sure it's implemented with a gyroscope in the controllers, but I think I'd like e.g. "motion control" better.

Reply:
I like “gyro” better, when I hear “motion control” I think of VR, Xbox Kinect and other irrelevant tech.